ArjunaJTS/integration/src/main/java/com/arjuna/ats/jbossatx/jts/TransactionManagerService.java

```diff
@@ -92,7 +100,7 @@ public void start(org.omg.CORBA.ORB theCorbaORB) throws Exception
     public void stop() {
         super.stop();
 
-        ORB.getInstance("jboss-atx").shutdown();
+        ORB.getInstance(ORB_NAME).shutdown();
```

Comment:
ORB.getInstance(ORB_NAME) returns our orbportability wrapper which calls our shutdown hooks. The external orb is set in ORBBase and it is that one that we want to avoid calling close on (since it may still be in use by other systems)